src/kernel/main.c

```diff
@@ -26,6 +26,7 @@
 #include <numbers.h>
 #include <ioapic.h>
 #include <keyboard.h>
+#include <log.h>
```

Comment:
mmm do you think that log.h can be confused with the mathematical log? 

should we call it logging.h or it should be fine this way too? 
What do you think?

Reply:
Haha I had this thought myself! I think in the context log.h is fine, as it's pretty unlikely we're using the mathematical log everywhere. Logging.h is more obvious though, I agree